package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "amazon-connect-chatjs",
-  "version": "1.3.2",
+  "version": "1.3.3",
```

Comment:
Does this need to be updated/regenerated in `package-lock.json`

Reply:
Thanks Spencer for catching this; updated!